### DISCUSSION TITLE
teleruttore Legrand- possibile funzionamento sempre eccitato ?

### DISCUSSION THREAD

**Discussion by gianry** [indomus/forum#2220] [GENERALE - Discussioni generiche]
Dovrei utilizzare uno Shelly EM per interrompere l'erogazione di energia della linea da 25 A quando si supera l'assorbimento di 16 A.
Posto di utilizzare il teleruttore Legrand 25 A con lo shelly questo dovrebbe lavorare principalmente sempre eccittato perchè ha contatti NA.
Secondo voi puo' durare nel tempo ? oppure c'è una soluzione migliore ?

grazie

**Comment by Maspar79** [comment]
Potresti usare un contattore con contatti NC, ed eccitare solo quando si superano i 16A.

Ciao

**Reply by gianry** [reply]
vero , ci ho pensato nel frattempo.. devo solo studiare se lo shelly EM configurato al contrario mi puo' portare altri inconvenienti.

grazie
